compass/package.json

```
@@ -3,23 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@kyma-project/luigi-client": "0.4.12",
     "@kyma-project/luigi-core": "0.4.12",
-    "fiori-fundamentals": "^1.7.0",
-    "fundamental-react": "^0.6.1",
-    "fundamental-styles": "^0.1.0",
     "jsoneditor": "^6.2.1",
     "react-ace": "^7.0.4",
-    "react-router-dom": "^5.0.1",
-    "recompose": "^0.30.0",
-    "styled-components": "^3.4.10",
     "uuid": "^3.3.2",
     "uuid-validate": "0.0.3",
     "xml-js": "^1.6.11"
   },
   "scripts": {
-    "start-live-server": "live-server --quiet --entry-file=index.html --mount=/luigi-core:./node_modules/@kyma-project/luigi-core public-luigi/",
-    "start": "concurrently \"PORT=8888 ../node_modules/.bin/react-scripts start\" \"npm run buildConfig:watch\" \"wait-on http://localhost:8888/ && npm run start-live-server\" ",
+    "start-live-server": "../node_modules/.bin/live-server --quiet --entry-file=index.html --mount=/luigi-core:./node_modules/@kyma-project/luigi-core public-luigi/",
+    "start": "../node_modules/concurrently/bin/concurrently.js \"PORT=8888 ../node_modules/.bin/react-scripts start\" \"npm run buildConfig:watch\" \"../node_modules/.bin/wait-on http://localhost:8888/ && npm run start-live-server\" ",
```

To be consistent with other commands: 
```suggestion
    "start": "../node_modules/.bin/concurrently.js \"PORT=8888 ../node_modules/.bin/react-scripts start\" \"npm run buildConfig:watch\" \"../node_modules/.bin/wait-on http://localhost:8888/ && npm run start-live-server\" ",
```